README.md

```diff
@@ -41,6 +42,7 @@
 
     ```elixir
     config :ueberauth, Ueberauth.Strategy.Amco.OAuth,
+      site: {System, :get_env, ["AMCO_IDP_URL"]},
```

Comment:
Is there an example in the read me that shows how we can configure this via runtime config and not environment variables?

Reply:
Runtime configuration file does not exclude env variables. The config option can come from env vars or direct string values which is our case. The runtime file generated by biran will have the config options as a strings.
IMO this makes clear to devs that the options should be secret.

Reply:
Right... but we should document how those strings should look and the fact that you can do that. I should be able to look at the code or documentation and easily figure out what values I need to place on the server in order for this to work as expected. Right now, I can't.

Reply:
Latest update addresses my concern